tap_zendesk/streams.py

```diff
@@ -308,12 +275,12 @@ def emit_sub_stream_metrics(sub_stream):
             self.update_bookmark(state, utils.strftime(generated_timestamp_dt))
 
             ticket.pop('fields') # NB: Fields is a duplicate of custom_fields, remove before emitting
-            should_yield = self._buffer_record((self.stream, ticket))
+            yield (self.stream, ticket)
```

Comment:
Yes because earlier it is required to identify the stream in function `_buffer_record` as there is login on the stream name, I don't see that logic now, do we still need this?

Reply:
added a comment regarding why we are yielding stream name and rec in the code